### PR TITLE
build: display background file loading from the fssync server

### PR DESCRIPTION
**Pull request by nicks**
Hello @landism,

Please review the following commits I made in branch nicks/buildkit:

f6c94c2f3a1082666531582f550848c9d347c84a (2022-01-11 17:49:18 -0500)
build: display background file loading from the fssync server

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
don't worry, this isn't that big! most of the diff is regenerating all the buildkit output and the golden files for testing

File: internal/build/testdata/TestBuildkitPrinter/generate-responses.sh [hunk 1]
```diff
@@ -4,7 +4,13 @@ set -ex
 
 DIR=$(realpath $(dirname $0))
 
-go install github.com/tilt-dev/tilt/cmd/buildkitapi
+#go install github.com/tilt-dev/tilt/cmd/buildkitapi
```
Comment (landism) [inline internal/build/testdata/TestBuildkitPrinter/generate-responses.sh:7]: 👀

File: internal/build/buildkit_printer.go [hunk 2]
```diff
@@ -37,25 +37,33 @@ type vertex struct {
 }
 
 const internalPrefix = "[internal]"
+const internalLoader = "[internal] load build context"
 const logPrefix = "  → "
 
 var stageNameRegexp = regexp.MustCompile(`^\[.+\]`)
 
 func (v *vertex) isInternal() bool {
```
Comment (landism) [inline internal/build/buildkit_printer.go:45]: ffti: `isInternal` is a bit inaccurate now - maybe something like `shouldPrint` or `shouldHide`?

**Review by landism — APPROVED** [review]

File: internal/build/testdata/TestBuildkitPrinter/generate-responses.sh (hunk 1 above)
Comment (nicks) [reply internal/build/testdata/TestBuildkitPrinter/generate-responses.sh]: oops

File: internal/build/buildkit_printer.go (hunk 2 above)
Comment (nicks) [reply internal/build/buildkit_printer.go]: ahhh good point

**Pull request merged**